jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonSamProjectWizard.kt

```diff
@@ -69,7 +69,7 @@ class SamDynamoDBCookieCutter : SamProjectTemplate() {
     override fun description() = message("sam.init.template.dynamodb_cookiecutter.description")
 
     override fun supportedZipRuntimes() = pythonTemplateRuntimes
-    override fun supportedImageRuntimes() = pythonTemplateRuntimes
+    override fun supportedImageRuntimes() = emptySet<LambdaRuntime>()
```

Comment:
Why are we assigning an emptyset?

Reply:
These templates were all Handler/Zip based, so they should not be shown when the selected type is Image so we are saying that for this template in  Image mode, no runtimes support it.